examples/md-flexible/src/Simulation.cpp

```diff
@@ -132,6 +132,12 @@ Simulation::Simulation(const MDFlexConfig &configuration,
   autopas::Logger::get()->set_level(_configuration.logLevel.value);
   _autoPasContainer->init();
 
+  // Throw an error if there is not more than one configuration to test in the search space but more than one tuning
+  // phase is requested
+  if (_autoPasContainer->getsearchSpaceIsTrivial() && _configuration.tuningPhases.value > 0) {
```

Comment:
```suggestion
  if (_autoPasContainer->getsearchSpaceIsTrivial() and _configuration.tuningPhases.value > 0) {
```
Both are equivalent. Suggestion is just for consistency with the rest of the project.

Reply:
Fixed!